### DISCUSSION TITLE
Is there a bug in the Cypress tests?

### DISCUSSION THREAD

**Discussion by jedkohjk** [reposense/RepoSense#2230] [Q&A] [answered]
I was running the frontend tests, and `chartView/chartView_mergeCommits.cy.js` and `chartView/chartView_optimiseTimeline.cy.js` failed. It appears that they record the [following commit](https://github.com/reposense/RepoSense/commit/a360c9edf08baffee4d934aab810e4e2a02e7a37) to have been on 2023-03-03 while on the dashboard I generate, it is on 2023-03-04. This results in several checks yielding different results. Is this an issue on my end, or is there an issue with the cypress test?

Screenshot of a failed test and the relevant html:
![Screenshot](https://github.com/reposense/RepoSense/assets/110604064/57884f1f-d19d-4e23-9ea6-99806a3a838d)

**Comment by ckcherry23** [comment, marked as answer]
This issue occurs for some users locally and is perhaps due to time zone differences. It is a known bug, but this Cypress test passes on the CI.
